### PR TITLE
Adds definitions for package redux-rsaa-middleware.

### PR DESCRIPTION
**Pull request by alessioprestileo**

### CODE REVIEW

**Comment by alessioprestileo** [comment]
I looked at the redux definitions for inspiration, and it looks like they only have tests for what is defined as 'declare export function'.
I don't have any functions among my definitions, so I'm not sure about how I should test them... Could you please point me to an example?

**Comment by alessioprestileo** [comment]
I tried to add a test file, does that make sense?

File: definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    finishedSendingAction?: RSAAAction,
+  };
+
+  declare export type RSAAState = Object;
+  
+  declare export type DispatchAPI<A> = (action: A) => A;
+  
+  declare export type Dispatch<A: { type: $Subtype<string> }> = DispatchAPI<A>;  
+  
+  declare export var RSAAMiddleware : Middleware<RSAAState, RSAAAction, Dispatch<RSAAAction>>;
```
Comment (GAntoine) [inline definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js]: Change to `module.exports`

File: definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+declare module 'redux-rsaa-middleware' {
+  import type { Middleware } from 'redux';
```
Comment (GAntoine) [inline definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js]: This won't work, as imports aren't currently functional. You can create a non-exported type here which mocks `Middleware`. Something like:

```
type Middleware = (req: http$IncomingMessage, res: http$ServerResponse, next: (err?: ?Error) => mixed) => mixed;
```

File: definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js (hunk 1 above)
Comment (AndrewSouthpaw) [reply definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js]: I'm curious, why was this necessary?

Comment (GAntoine) [reply definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js]: Well it wasn't, that was my sleepy state talking. I just changes it back in a commit. -.-

**Pull request merged**